marklogic-client-api/src/main/java/com/marklogic/client/expression/PlanBuilderBase.java

```diff
@@ -791,4 +791,6 @@ interface ModifyPlanBase {
      */
     interface PreparePlanBase {
     }
+
+  public abstract PlanBuilder.AccessPlan fromDocUris(CtsQueryExpr querydef, String qualifier);
```

Comment:
I think this should be with the other "from" methods at the top of `PlanBuilderBase` - I'd put it after `fromLiterals(Map)` at line 153 and before `jsonObject`. 

Also, we need javadoc comments for this, correct? I'm wondering what `qualifier` in particular is - if it's the same as `qualifierName` in the other `from` methods, then it should be renamed to `qualifierName` for consistency.